tests/timeseries/test_moving_window.py

```diff
@@ -66,6 +66,18 @@ def init_moving_window(
     return window, lm_tx
 
 
+def dt(i: int) -> datetime:  # pylint: disable=invalid-name
+    """Create datetime objects from indices.
+
+    Args:
+        i: Index to create datetime from.
+
+    Returns:
+        Datetime object.
+    """
+    return datetime.fromtimestamp(i, tz=timezone.utc)
+
+
 async def test_access_window_by_index() -> None:
```

Comment:
I wonder if that's the correct function to add the tests. This was supposed to access the window by integer index, but the tests are accessing by datetime object.

Reply:
The newly added datetime slice tests were added in `test_access_window_by_ts_slice`. Then a single failure test to for int slices in `test_access_window_by_int_slice`